### DISCUSSION TITLE
After Vercel deployment, it doesn't show css for Astro Framework.

### DISCUSSION THREAD

**Discussion by NonsensicalInsane** [vercel/community#6613] [Help]
### Summary

When I execute "npm run dev" or "npx astro dev" in my local computer, my website show all features without error after "npm run build" or "npx astro build". However, when I deploy to Vercel, Vercel doesn't show any css in my website.  I don't why this is happened, can anyone help me on this?
This website is builded on Astro Framework.

### Steps to Reproduce

This page is in my local computer (When I execute "npm run dev" after building):
![240424_03h02m44s_screenshot](https://github.com/vercel/community/assets/48383391/acf3a109-3d42-48d4-a9d8-39e30c7100b6)

This is after pushing Github and deploying Vercel.:
![240424_03h06m15s_screenshot](https://github.com/vercel/community/assets/48383391/0b13e99a-e017-4307-b00e-e9fb2c39d729)

This is the Github Repo: https://github.com/NonsensicalInsane/furkaneyazici
This is the website: https://furkaneyazici.com

**Comment by amyegan** [comment]
Hey @NonsensicalInsane. The site styles seem to load correctly for me at https://furkaneyazici.com/. Did you already find a solution?

<img width="1253" alt="screenshot of web page with CSS styles applied" src="https://github.com/vercel/community/assets/8247105/175629e2-bab7-4b18-bcbe-270307fa55aa">

**Reply by NonsensicalInsane** [reply]
Hi @amyegan, thank you for asking this, and sorry for the late reply - and I didn't see the notification. I forgot to update you on the current situation. I actually deleted the Vercel adapter in the astro.config.mjs. First, I built the local site in the dist folder using "npm run build". Afterwards, I uploaded this latest version and the problem was temporarily solved. I still don’t know the reason. 

Later on, I didn’t build and just uploaded the entire site without building and without the adapter. Vercel built it by itself and published this version.  Right now, the site seems fine, but as I told you, the problem is solved only temporarily because I cannot use adapter functions, like analytics.
